docs/api.yaml

```diff
@@ -98,7 +134,8 @@ components:
         quotes:
           type: array
           items:
-            $ref: '#/components/schemas/Quote'
```

Comment:
Did we not want to have the other data in a `Quote` returned?

Reply:
Do you think we should have more stuff? Previously it was actually just a list of strings so I was fixing this. But I'm open to including more details if you think it's worthwhile.

Reply:
I was just thinking if we would need to expand it in the future if a `Quote` holds more than just content. But I think this is fine for now and we'll just update this when we need to